deebot_client/commands/json/common.py

```diff
@@ -88,13 +89,21 @@ class ExecuteCommand(CommandWithMessageHandling, ABC):
     """Command, which is executing something (ex. Charge)."""
 
     @classmethod
-    def _handle_body(cls, event_bus: EventBus, body: dict[str, Any]) -> HandlingResult:
+    def _handle_body(
+        cls, event_bus: EventBus, body: dict[str, Any] | str
+    ) -> HandlingResult:
         """Handle message->body and notify the correct event subscribers.
 
         :return: A message response
         """
+        # Success events from the XML api looks like <ctl ret='ok'/>
+        if isinstance(body, str):
```

Comment:
This code here should be moved to the xml folder. As I was not sure, which code parts you need I left it in json folder for now.